2_accept_offer.py

```diff
@@ -93,7 +93,11 @@ def process_offer(offer, audit):
     + locale.format("%.8f", Decimal(offer_currency_quantity) / COIN, True) + " " + offer_currency_code + " for " \
     + locale.format("%.8f", Decimal(ask_currency_quantity) / COIN, True) + " " + ask_currency_code
 
-  # TODO: Prompt the user for whether the trade is acceptable
+  answer = raw_input("Do you want to continue with the trade? 'Y' to continue, 'N' to quit.  ").lower()
+  while answer not in ['y', 'yes', 'n', 'no']:
+    answer = raw_input("Not recognised. 'Y' to continue, 'N' to quit.  ").lower()
+  if answer != 'y' or answer != 'yes':
```

Comment:
Yes it should and my test ver has that - meh I'm not good at this. I should have copied it from my test file rather than retyped it! I'll try and claim I am ill right now and a lot stressed but I still should have spotted it cos it's obvious.

I retyped it btw so I wouldn't mess up the indenting which also is a bit stupid!

Sorry! I can't believe I messed up a stupid simple thing like a logic test (am mortified).


> On 7 Jan 2015, at 22:54, Ross Nicoll <notifications@github.com> wrote:
> 
> In 2_accept_offer.py:
> 
> > @@ -93,7 +93,11 @@ def process_offer(offer, audit):
> >      + locale.format("%.8f", Decimal(offer_currency_quantity) / COIN, True) + " " + offer_currency_code + " for " \
> >      + locale.format("%.8f", Decimal(ask_currency_quantity) / COIN, True) + " " + ask_currency_code
> >  
> > -  # TODO: Prompt the user for whether the trade is acceptable
> > +  answer = raw_input("Do you want to continue with the trade? 'Y' to continue, 'N' to quit.  ").lower()
> > +  while answer not in ['y', 'yes', 'n', 'no']:
> > +    answer = raw_input("Not recognised. 'Y' to continue, 'N' to quit.  ").lower()
> > +  if answer != 'y' or answer != 'yes':
> Shouldn't this be "and"? As is, it wants it to be both 'y' and 'yes' in order to not exit.
> 
> \
> Reply to this email directly or view it on GitHub.
>